doc/nrf/releases/release-notes-2.3.0.rst

```diff
@@ -0,0 +1,1039 @@
+  * Updated the sample, so that it waits for RRC idle mode before requesting neighbor cell measurements.
+
+* :ref:`lwm2m_client` sample:
+
+  * Added:
+
+    * Support for nRF7002 EK shield and Wi-Fi based location.
+    * Location events and event handlers.
+
+  * Updated:
+
+    * The sensor module has been simplified.
+      It does not use application events, filtering, or configurable periods anymore.
+    * Replaced deprecated LwM2M API calls with calls to new functions.
+    * Enabled LwM2M queue mode and updated documentation accordingly.
+    * Moved configuration options from :file:`overlay-queue.conf` to default configuration :file:`prj.conf`.
+    * Removed :file:`overlay-queue.conf`.
+    * Enabled :kconfig:option:`CONFIG_LTE_LC_TAU_PRE_WARNING_NOTIFICATIONS` configuration option.
+
+* :ref:`http_application_update_sample` sample:
+
+  * Added support for the :ref:`liblwm2m_carrier_readme` library.
+
+* :ref:`nrf_cloud_mqtt_multi_service` sample:
+
+  * Added:
+
+    * MCUboot child image files to properly access external flash on newer nRF9160DK versions.
+    * An :file:`overlay_mcuboot_ext_flash.conf` file to enable MCUboot use of external flash.
+    * Sending an alert to the cloud on boot and when a temperature limit is exceeded.
+
+  * Updated:
+
+    * The sample now uses a partition in external flash for full modem FOTA updates.
+
+* :ref:`nrf_cloud_rest_device_message` sample:
+
+  * Added sending an alert to nRF Cloud on boot.
+
+* :ref:`slm_shell_sample` sample:
+
+  * Added:
+
+    * Sample for nRF52 and nRF53 Series devices to send AT commands to nRF9160 SiP from shell.
+
+* Removed:
+
+  * Multicell location sample because of the deprecation of the Multicell location library.
+    Relevant functionality is available through the :ref:`lib_location` library.
+  * nRF9160: Simple MQTT sample.
+    This is now replaced by a new :ref:`mqtt_sample` sample that supports Wi-Fi and LTE connectivity.
+
+* :ref:`nrf_cloud_rest_fota` sample:
+
+  * Updated:
+
+    * Device status information, including FOTA enablement, is now sent to nRF Cloud when the device connects.
+    * Removed user prompt and button press handling for FOTA enablement.
+    * The sample now uses a partition in external flash for full modem FOTA updates.
+
+* :ref:`azure_fota_sample` sample:
+
+  * The sample now uses the logging subsystem for console output.
+
+* :ref:`azure_iot_hub` sample:
+
+  * The sample now uses the logging subsystem for console output.
+
+* :ref:`aws_iot` sample:
+
+  * The sample now uses the logging subsystem for console output.
+
+Peripheral samples
+------------------
+
+* :ref:`radio_test` sample:
+
+  * Added support for the nRF7002 DK.
+  * Fixed sample building with support for the Skyworks front-end module.
+  * Updated the documentation to clarify that this sample is dedicated for the short-range radio (Bluetooth LE, IEEE 802.15.4, and proprietary modes).
```

Comment:
```suggestion
  * Updated the documentation to clarify that this sample is dedicated for the short-range radio (Bluetooth LE, IEEE 802.15.4, and proprietary modes).
  * Fixed sample building with support for the Skyworks front-end module.
```

Reply:
Any reason for this? I'd say doc should be mentioned last.